proto/osmosis/protorev/v1beta1/protorev.proto

```diff
@@ -0,0 +1,57 @@
+syntax = "proto3";
+package osmosis.protorev.v1beta1;
+
+import "cosmos_proto/cosmos.proto";
+import "gogoproto/gogo.proto";
+
+import "cosmos/base/v1beta1/coin.proto";
+
+option go_package = "github.com/osmosis-labs/osmosis/v13/x/protorev/types";
+
+// TokenPairArbRoutes tracks all of the hot routes for a given pair of tokens
+message TokenPairArbRoutes {
+  option (gogoproto.equal) = true;
+
+  // Stores all of the possible hot paths for a given pair of tokens
+  repeated Route arb_routes = 1;
+  // Token denomination of the first asset
+  string token_a = 2;
+  // Token denomination of the second asset
+  string token_b = 3;
```

Comment:
Could we consider better nomenclature for these? Perhaps in/out?

Reply:
also nit: consistent Proto fields using `_` (snake-case).

Reply:
@davidterpay @NotJeremyLiu my understanding was there wasn't a consistent "in/out" associated with these?

Reply:
We made a design decision to make the route building logic cleaner and constraining the type of data that is inputted to protorev instead of having reversal logic built in as well. With this design, anytime we see a swap with a -> b we can instantly find all hot routes that are valid without having to do any additional logic (or compute). If we also want hot routes for b -> a, we have to pass in a additional hot routes (i.e. the hot routes are always ordered and directed in the correct direction of the trade).